pkg/cmd/server/start/start_node.go

```diff
@@ -280,8 +274,13 @@ func StartNode(nodeConfig configapi.NodeConfig) error {
 
 	config.EnsureVolumeDir()
 	config.EnsureDocker(docker.NewHelper())
+	plugin, endpointFilter := InitSDNPlugin(config, nodeConfig)
```

Comment:
plugin and endpointFilter could be added to the `*kubernetes.NodeConfig` that `BuildKubernetesNodeConfig` returns